CONTRIBUTING.md

```diff
@@ -35,12 +35,24 @@ Most of Phase's code is under the MIT license, though some paid feature restrict
 Any third party components incorporated into our code are licensed under the original license provided by the applicable component owner.
 
 ## Setup local development environment
-
 ### Dev server with hot reload
-
-1. Create a `.env.dev` file with `cp .env.dev.example .env.dev` and add atleast one OAuth provider.
-2. `docker-compose -f dev-docker-compose.yml up`.
-3. The Console is now running at `https://localhost` with HMR.
+1. Create a **.env.dev** file using
+    ```
+    cp .env.dev.example .env.dev
+    ```
+2. Add atleast one OAuth provider in your **.env.dev**. Follow the [docs](https://docs.phase.dev/self-hosting/configuration/envars)
+2. Check **dev-docker-compose.yml** file is populating correctly with env variables by running 
+    ```
+    docker compose -f dev-docker-compose.yml --env-file .env.dev config
+    ```
+3. If all the configs are correct, start the cotainers using 
+    ```
+    docker-compose -f dev-docker-compose.yml --env-file .env.dev up
+    ```
+4. The Console is now running at <http://localhost> with HMR(Hot Module Reload)
+   You can also connect using https protocol - <https://localhost>
+   >**Note :** It might show **Your connection isn't private**. 
+   You can ignore the message and continue using localhost
```

Comment:
```suggestion
4. The Console is now running at <https://localhost> with HMR(Hot Module Reload) and a self-signed certificate.
   >**Note : Your browser might warn you about the self-signed certificate. You can safely accept the certificate and proceed. 
```

Updated this instruction to to use `https` by default, as some browsers will automatically upgrade you anyway. Also made the instruction for swallowing the self-signed cert more generic, as developers might be using different browser which display this warning differently. 

Also, @Jay-Iam lets update this to use `http` once #33  is resolved